NitroxClient/GameLogic/ItemContainers.cs

```diff
@@ -47,6 +48,9 @@ public void BroadcastItemAdd(Pickupable pickupable, Transform containerTransform
             
             if (packetSender.Send(new EntityReparented(itemId, ownerId)))
             {
+                var iOxygenSource = pickupable.gameObject.GetComponent<IOxygenSource>();
```

Comment:
We don't use var at all.
It seems you don't get visual feedback from your IDE for this. We have a `.editorconfig` file in the project which should setup out code style for you... :thinking: